constance/backends/redisd.py

```diff
@@ -23,7 +24,7 @@ def __init__(self):
             try:
                 import redis
             except ImportError:
-                raise ImproperlyConfigured('The Redis backend requires redis-py to be installed.')
+                raise ImproperlyConfigured('The Redis backend requires redis-py to be installed.') from None
```

Comment:
raises from None actually just silent linter. I suppose some linter bug. Correct usage:
```
except ImportError as exc:
    raise ImproperlyConfigured('The Redis backend requires redis-py to be installed.') from exc
```

Reply:
Here from exc does not provide for user more readable exception.  If import error occurred you actually do not need more than "The Redis backend requires redis-py to be installed"